app/controllers/declaration/RoutingCountriesSummaryController.scala

```diff
@@ -122,7 +122,7 @@ class RoutingCountriesSummaryController @Inject()(
 
   def submitChangeCountry(mode: Mode, countryToChange: String): Action[AnyContent] = (authenticate andThen journeyType).async { implicit request =>
     val cachedCountries = request.cacheModel.locations.routingCountries
-    val countriesForValidation = cachedCountries.filter(_ == countryToChange)
+    val countriesForValidation = cachedCountries.filterNot(_ == countryToChange)
```

Comment:
So countriesForValidation are the existing countries in the cache except the one you want to change.  Makes sense.

Reply:
Yes, to check duplication in easier way